### ISSUE TITLE
should allow to cd to the specified location after $ vagrant ssh

### ISSUE THREAD

**Issue by hoatle** [teracyhq/dev#296]
This is an improvement for experience and time saving.

We usually $ ws after $ vagrant ssh (a lot of times).

By default, after $ vagrant ssh, $ ws should be run right after that.

And users can configure the hooked command to run right after $ vagrant ssh

**Comment by hoatle** [comment]
at the moment, we can use alias to go to the right project path instead of $ ws

for example:

```
{
  "provisioners": [{
    "_id": "0",
    "json": {
      "teracy-dev": {
        "aliases": [{
          "_id": "1",
          "name": "mp",
          "command": "cd ~/workspace/my-project",
          "action": "add" // one of add, remove. Default: add.
          // See more: https://github.com/customink-webops/magic_shell/blob/master/resources/alias.rb
        }]
      }
    }
  }]
}
```

This helps a bit, instead of:

```
$ vagrant ssh
$ ws
$ cd my-project
```

to:

```
$ vagrant ssh
$ mp
```

The next step for this issue is that:

```
$ vagrant ssh
```

and then we're at `~/workspace/my-project` immediately.

**Comment by phuonglm** [comment]
use .bashrc for that, just adding. But let define how to make config convenient for user.
just add cd ~/workspace/iorad to the end of .bashrc then we got this.
![screen shot 2017-05-18 at 9 26 59 am](https://cloud.githubusercontent.com/assets/646429/26183962/370d621a-3bac-11e7-97b0-14b6b3cb7880.png)

**Comment by phuonglm** [comment]
Or we can make ws command more useful like `ws iorad` then it'll search the workspace directory for iorad and cd to that.

**Comment by hoatle** [comment]
closed as it's irrelevant for v0.6.0 core, this can be extended instead.
